coreos-release-note-generator/README.md

```diff
@@ -0,0 +1,82 @@
+# coreos-release-note-generator
```

Comment:
Should we rename to `coreos-release-notes-generator`?

Reply:
Agreed :+1: